src/widget/systemtrayicon.cpp

```diff
@@ -26,8 +26,9 @@
 #include <QDebug>
 #include "src/persistence/settings.h"
 
-SystemTrayIcon::SystemTrayIcon()
+SystemTrayIcon::SystemTrayIcon(Widget *parent)
```

Comment:
@agilob Uhm ... we seem to have a private implementation here ... so I think that ...
a) `parent` member should be moved there
b) private class should be initialized here

Right?